build.gradle.kts

```diff
@@ -64,6 +64,7 @@ val versionImmutables = "2.9.0"
 val versionJackson = "2.13.3"
 val versionJacksonSpark3 = "2.13.3"
 val versionJacoco = "0.8.8"
+val versionJakartaAnnotationApi = "1.3.5"
 val versionJakartaEnterpriseCdiApi = "2.0.2"
 val versionJakartaValidationApi = "2.0.2"
```

Comment:
do these versions need to have any relation to each-other or can they be independently upgraded?

Reply:
:shrug: